http-server/src/main/java/io/strimzi/admin/Main.java

```diff
@@ -42,4 +53,36 @@ public static void main(final String[] args) {
 
         return promise.future();
     }
+
+    private static Map envVarsToAdminClientConfig() throws Exception {
+        Map envConfig = System.getenv().entrySet().stream().filter(entry -> entry.getKey().startsWith(PREFIX))
+                .collect(Collectors.toMap(entry -> entry.getKey().substring(PREFIX.length()), Map.Entry::getValue));
```

Comment:
why not just filter and not removing the prefix as well? I think that removing is misleading if not really needed. I guess it comes from the bridge but there we have different components (http, amqp, ..) in this case we are talking about Admin client configuration.

Reply:
I would stick having the prefix but it seems you removed it definitely :-(